.vscode/launch.json

```diff
@@ -30,6 +30,22 @@
         "${workspaceFolder}/testdata/invalid.zip",
       ]
     },
+    {
+      "name": "Debug sourcemapdiff (panel)",
+      "type": "go",
+      "request": "launch",
+      "mode": "auto",
+      "program": "${workspaceFolder}/pkg/cmd/sourcemapdiff",
+      "cwd": "${workspaceFolder}",
+      "args": [
+        "-nonInteractive",
+        "y",
+        "-archiveUri",
+        "https://github.com/briangann/grafana-gauge-panel/releases/download/v2.0.0/briangann-gauge-panel-2.0.0.zip",
+        "-sourceCodeUri",
+        "https://github.com/briangann/grafana-gauge-panel/tree/v2.0.0",
+      ]
+    },
```

Comment:
Nit: maybe we can also take the yarn v3 source maps from this plugin and add a separate test case for it in `pkg/analysis/passes/jssourcemap/testdata` and `pkg/analysis/passes/jssourcemap/jssourcemap_test.go`, so this new case runs in CI as well?

Reply:
i think that's a great idea, will implement it